RebelAllianceBank/Classes/CardAccount.cs

```diff
@@ -3,11 +3,14 @@ namespace RebelAllianceBank.Classes
 {
     public class CardAccount : IBankAccount
     {
+        public int ID { get; set; }
+        public string UserId { get; set; }
+        public int AccountType { get; set; }
```

Comment:
Maybe add these to the constructor?

Reply:
Added to the constructor. Left ID as it is, as we don't know how to implement just yet.